jsonschema-core.md

```diff
@@ -816,74 +778,41 @@ circumstances.
 
 ### Loading Instance Data
 
-While none of the vocabularies defined as part of this or the associated
-documents define a keyword which may target and/or load instance data, it is
-possible that other vocabularies may wish to do so.
+While none of the keywords defined as part of this or the associated
+documents define a keyword which target and/or load instance data, it is
+possible that extensions may wish to do so.
 
 Keywords MAY be defined to use JSON Pointers or Relative JSON Pointers to
 examine parts of an instance outside the current evaluation location.
 
 Keywords that allow adjusting the location using a Relative JSON Pointer SHOULD
 default to using the current location if a default is desireable.
 
-## The JSON Schema Core Vocabulary {#core}
-
-Keywords declared in this section, which all begin with "$", make up the JSON
-Schema Core vocabulary. These keywords are either required in order to process
-any schema or meta-schema, including those split across multiple documents, or
-exist to reserve keywords for purposes that require guaranteed interoperability.
+## The JSON Schema Core Keywords {#core}
 
-The Core vocabulary MUST be considered mandatory at all times, in order to
-bootstrap the processing of further vocabularies. Meta-schemas that use the
-[`$vocabulary`](#vocabulary) keyword to declare the vocabularies in use MUST
-explicitly list the Core vocabulary, which MUST have a value of true indicating
-that it is required.
+Keywords declared in this section, which all begin with "$", are essential to
+processing JSON Schema. These keywords inform implementations how to process any
+schema or meta-schema, including those split across multiple documents, or exist
+to reserve keywords for purposes that require guaranteed interoperability.
 
-The behavior of a false value for this vocabulary (and only this vocabulary) is
-undefined, as is the behavior when `$vocabulary` is present but the Core
-vocabulary is not included. However, it is RECOMMENDED that implementations
-detect these cases and raise an error when they occur. It is not meaningful to
-declare that a meta-schema optionally uses Core.
+Support for these keywords MUST be considered mandatory at all times in order to
+bootstrap the processing of further keywords.
 
-Meta-schemas that do not use `$vocabulary` MUST be considered to require the
-Core vocabulary as if its IRI were present with a value of true.
+The "$" prefix is reserved for use by this specification. Extensions MUST NOT
+define new keywords that begin with "$".
 
-The current IRI for the Core vocabulary is:
-`https://json-schema.org/draft/next/vocab/core`.
+### Meta-Schemas
 
-The current IRI for the corresponding meta-schema is:
-`https://json-schema.org/draft/next/meta/core`.
+Meta-schemas are used to inform an implementation how to interpret a schema.
+Every schema has a meta-schema, which can be explicitly declared using the
+`$schema` keyword.
```

Comment:
I think this section was effectively the introduction to and definition of the term "dialect". That ends up getting lost in the changes. Do we want to do something to include that defines "dialect"?

Reply:
I'm not sure.  I wonder if that's something that needs to be re-introduced with the proposal.

Reply:
I think the concept of a dialect should stay. The concept isn't dependent on the vocabulary system. I explain further in my latest review.

Reply:
How do you want to define "dialect"?  Is it merely the collection of keywords?  Open to suggestions.